src/migrations/20181205173036_added_todo.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('todos', table => {
+    table.increments();
+    table
+      .timestamp('created_at')
+      .notNull()
+      .defaultTo(knex.raw('now()'));
+    table.timestamp('updated_at').notNull();
+    table.string('title').notNull();
+    table.string('status').notNull();
+    table.string('username').notNull();
+    table.string('uuid').notNull().unique();
```

Comment:
What does `uuid` mean?

Reply:
unique id for each todo. Kaile kai same todo vaye uniquely identify garna haleko